packages/demo-html/public/sidetab-html.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Static HTML Demo</title>
+    <style>
+      #wrapper {
+        width: 400px;
+        height: 580px;
+        margin: 0 auto;
+      }
+    </style>
+  </head>
+  <body>
+    <script src="./lib/embed-next.js"></script>
```

Comment:
Does sidetab support HTM syntax? Eg. `<div data-tf-sidetab="<form-id>"></div>`? How does this page render the sidetab embed?